packages/cli/src/help/index.ts

```diff
@@ -0,0 +1,72 @@
+import { Help } from '@oclif/core'
```

Comment:
I assume some of this code is taken from the base class. If I'm correct, it's better to reuse that code (e.g, inherit) than to copy it.

Reply:
Hey yes you are correct, some part of it is pasted from the base class itself.
So should I use super or something to achieve that?